routes/index.js

```diff
@@ -117,8 +117,18 @@ function routes(app) {
 
       if (job) {
         // bind data to page
+        const title = _.get(job, 'data.title');
```

Comment:
This feels unnecessary given that the only thing we're modifying here is the link produced.
We can do all of this (lines 120-131) in the view itself (`views/pages/view-result.html`).